generators/common/templates/quarkus/README.md.ejs

```diff
@@ -102,30 +102,40 @@ To log in to your app, you'll need to have [Keycloak](https://keycloak.org) up a
 docker-compose -f src/main/docker/keycloak.yml up
 ```
 
-The security settings in `src/main/resources/config/application.yml` are configured for this image.
+The security settings in `src/main/resources/config/application.properties` are configured for this image.
 
 ```properties
 quarkus.oidc.enabled=true
 quarkus.oidc.auth-server-url=http://localhost:9080/auth/realms/jhipster/
 quarkus.oidc.client-id=web_app
 quarkus.oidc.credentials.secret=web_app
-quarkus.oidc.authentication.scopes=profile
-quarkus.oidc.application-type=web-app
+quarkus.oidc.authentication.scopes=profile,address,email,address,phone,offline_access
+quarkus.oidc.application-type=hybrid
+quarkus.oidc.authentication.cookie-path=/
+quarkus.oidc.authentication.redirect-path=/login/oauth2/code/oidc
+quarkus.oidc.authentication.restore-path-after-redirect=false
+
+jhipster.oidc.logout-url=http://localhost:9080/auth/realms/jhipster/protocol/openid-connect/logout
 ```
 
 ### Okta
 
 If you'd like to use Okta instead of Keycloak, you'll need to change a few things. First, you'll need to create a free developer account at <https://developer.okta.com/signup/>. After doing so, you'll get your own Okta domain, that has a name like `https://dev-123456.okta.com`.
 
-Modify `src/main/resources/config/application.yml` to use your Okta settings.
+Modify `src/main/resources/config/application.properties` to use your Okta settings.
 
 ```properties
 quarkus.oidc.enabled=true
 quarkus.oidc.auth-server-url=https://{yourOktaDomain}/oauth2/default
 quarkus.oidc.client-id={clientId}
 quarkus.oidc.credentials.secret={clientSecret}
-quarkus.oidc.authentication.scopes=profile
-quarkus.oidc.application-type=web-app
+quarkus.oidc.authentication.scopes=profile,address,email,address,phone,offline_access
+quarkus.oidc.application-type=hybrid
+quarkus.oidc.authentication.cookie-path=/
+quarkus.oidc.authentication.redirect-path=/login/oauth2/code/oidc
+quarkus.oidc.authentication.restore-path-after-redirect=false
+
+jhipster.oidc.logout-url=https://{yourOktaDomain}/oauth2/default/v1/logout
```

Comment:
I think it might be better to just reduce this to the properties that need to change, so:

```
quarkus.oidc.auth-server-url=https://{yourOktaDomain}/oauth2/default
quarkus.oidc.client-id={clientId}
quarkus.oidc.credentials.secret={clientSecret}
jhipster.oidc.logout-url=https://{yourOktaDomain}/oauth2/default/v1/logout
```

Related: in the default `application.properties`, it has:

```
%dev.quarkus.oidc.client-id=web_app
%dev.quarkus.oidc.credentials.secret=web_app
```

Why is `%dev` necessary here? Will it apply to prod profiles too?

Reply:
> I think it might be better to just reduce this to the properties that need to change, so:
👍 
> Why is %dev necessary here? Will it apply to prod profiles too?
https://quarkus.io/guides/config#configuration-profiles
I don't have any particular conviction yet on profile management with Quarkus.
My habit is to enforce explicitness that's why I used `%dev` it make it explicit for the developer he/she has to set a value before moving to prod.
Let's try your proposal it's consistent with JHipster upstream configuration.